src/events/aws_sdk_v3.js

```diff
@@ -56,6 +57,71 @@ const SNSv3EventCreator = {
     },
 };
 
+const SQSEventCreator = {
+    /**
+     * Updates an event with the appropriate fields from a SQS command
+     * @param {string} operation the operation we wrapped.
+     * @param {Command} command the wrapped command
+     * @param {proto.event_pb.Event} event The event to update the data on
+     */
+    requestHandler(operation, command, event) {
+        const parameters = command.input || {};
+        const resource = event.getResource();
+
+        if ('QueueUrl' in parameters) {
+            resource.setName(`${parameters.QueueUrl.split('/').pop()}`);
```

Comment:
I think you should try catch in case of unexpected URL

Reply:
Updated to something like this:
if(data !== null && data !== '') {
   // do something
}